parsers/s01-parse/fulljackz/proxmox-logs.yaml

```diff
@@ -0,0 +1,30 @@
+#debug: true
+name: fulljackz/proxmox-logs
+description: "Parse proxmox logs for bruteforce attempts"
+filter: "evt.Parsed.program == 'pvedaemon'"
+onsuccess: next_stage
+pattern_syntax:
+  PVE_AUTH_FAIL: 'authentication failure; rhost=%{IPV6:client_ip6}:%{IPV4:client_ip4} user=%{USERNAME:source_user}@%{WORD:realm} msg=Authentication failure'
+  PVE_NOSU_USER: 'authentication failure; rhost=%{IPV6:client_ip6}:%{IPV4:client_ip4} user=%{USERNAME:source_user}@%{WORD:realm} msg=no such user'
```

Comment:
did you had the chance to try it with "real" ipv4/ipv6 ? This format looks like you're accessing ipv6 address from an ipv4 address. [`%{IP}`](https://doc.crowdsec.net/docs/next/parsers/patterns#ip) might do the trick ! Please let me know

Reply:
I replaced `%{IPV6:client_ip6}:%{IPV4:client_ip4}` with `%{IP:client_ip}` and it seems ok.
I tried to connect using ipv6 and added one into log for testing.

So now I have `results["s01-parse"]["fulljackz/proxmox-logs"][12].Evt.Meta["source_ip"] == "::ffff:172.21.10.2"` 
Is this source_ip format is ok and well known by the bouncer ?